recipes/glinfo-rs/build.sh

```diff
@@ -0,0 +1,7 @@
+set -e
+
+export CARGO_HOME=$PWD/.cargo
+export CARGO_TERM_COLOR=always
+
+cargo build --release
+cp target/release/glinfo $PREFIX/bin
```

Comment:
```
-rwxr-xr-x 2 conda conda 3795272 Jul 26 01:07 $PREFIX/bin
```

Seems like there is no `$PREFIX/bin` directory, so it is copying `target/release/glinfo` as a ``$PREFIX/bin` *file*.

Try this:

```suggestion
mkdir -p $PREFIX/bin
cp --verbose target/release/glinfo $PREFIX/bin
```